include/deal.II/base/point.h

```diff
@@ -195,6 +195,17 @@ class Point : public Tensor<1, dim, Number>
   DEAL_II_CUDA_HOST_DEV Number &
                         operator()(const unsigned int index);
 
+  /**
+   * Assignment operator from Tensor<1, dim, Number> with different underlying
+   * scalar type. This obviously requires that the @p OtherNumber type is
+   * convertible to @p Number.
+   *
+   * @note This function can also be used in CUDA device code.
```

Comment:
Ah, I missed that there is still this comment. Would you removing it?
```suggestion
```

Reply:
Sorry, I forgot to remove that comment. Now it is done!